### ISSUE TITLE
App crashes on android API 21 and 22

### ISSUE THREAD

**Issue by ujjwalnitw** [airbnb/lottie-android#1131]
Hello,
The app crashes on when the animation is run on Android devices with API 21 and 22. 
Could you please help me on this? I am using 2.7.0 lottie version.
I have also attached zip file consiting of json and image.
Following  are the logs

`02-22 14:42:11.975 5404-5404/com.example.ujjwalgoyal.lottiesample E/art: Tried to mark 0x3 not contained by any spaces
    Attempting see if it's a bad root
    Found invalid root: 0x3 
    Type=RootJavaFrame thread_id=1 location=Visiting method 'void com.airbnb.lottie.model.layer.BaseLayer.setProgress(float)' at dex PC 0x0060 (native PC 0xa4611080) vreg=2
02-22 14:42:11.975 5404-5404/com.example.ujjwalgoyal.lottiesample A/art: art/runtime/gc/collector/mark_sweep.cc:381] Can't mark invalid object
02-22 14:42:12.001 5404-5404/com.example.ujjwalgoyal.lottiesample A/art: art/runtime/runtime.cc:289] Runtime aborting...
    art/runtime/runtime.cc:289] Aborting thread:
    art/runtime/runtime.cc:289] "main" prio=5 tid=1 Runnable
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x7310f000 self=0xb4025800
    art/runtime/runtime.cc:289]   | sysTid=5404 nice=0 cgrp=default sched=0/0 handle=0xb77ddea0
    art/runtime/runtime.cc:289]   | state=R schedstat=( 0 0 0 ) utm=173 stm=16 core=1 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xbf3b6000-0xbf3b8000 stackSize=8MB
    art/runtime/runtime.cc:289]   | held mutexes= "abort lock" "mutator lock"(shared held)
    art/runtime/runtime.cc:289]   native: #00 pc 00005d03  /system/lib/libbacktrace_libc++.so (UnwindCurrent::Unwind(unsigned int, ucontext*)+83)
    art/runtime/runtime.cc:289]   native: #01 pc 00003051  /system/lib/libbacktrace_libc++.so (Backtrace::Unwind(unsigned int, ucontext*)+33)
    art/runtime/runtime.cc:289]   native: #02 pc 003c9ea7  /system/lib/libart.so (art::DumpNativeStack(std::__1::basic_ostream<char, std::__1::char_traits<char> >&, int, char const*, art::mirror::ArtMethod*)+135)
    art/runtime/runtime.cc:289]   native: #03 pc 00393212  /system/lib/libart.so (art::Thread::Dump(std::__1::basic_ostream<char, std::__1::char_traits<char> >&) const+306)
    art/runtime/runtime.cc:289]   native: #04 pc 00377944  /system/lib/libart.so (art::AbortState::Dump(std::__1::basic_ostream<char, std::__1::char_traits<char> >&)+644)
    art/runtime/runtime.cc:289]   native: #05 pc 00377ead  /system/lib/libart.so (art::Runtime::Abort()+125)
    art/runtime/runtime.cc:289]   native: #06 pc 000ec704  /system/lib/libart.so (art::LogMessage::~LogMessage()+1876)
    art/runtime/runtime.cc:289]   native: #07 pc 001cb8d1  /system/lib/libart.so (bool art::gc::accounting::HeapBitmap::AtomicTestAndSet<art::gc::collector::MarkSweepMarkObjectSlowPath>(art::mirror::Object const*, art::gc::collector::MarkSweepMarkObjectSlowPath const&)+625)
    art/runtime/runtime.cc:289]   native: #08 pc 00084e4a  /system/lib/libart.so (art::gc::collector::MarkSweep::MarkObjectParallel(art::mirror::Object const*) (.part.121)+38)
    art/runtime/runtime.cc:289]   native: #09 pc 001cf74a  /system/lib/libart.so (art::gc::collector::MarkSweep::MarkRootParallelCallback(art::mirror::Object**, void*, art::RootInfo const&)+138)
    art/runtime/runtime.cc:289]   native: #10 pc 0039a015  /system/lib/libart.so (art::ReferenceMapVisitor<art::RootCallbackVisitor>::VisitQuickFrame()+1269)
    art/runtime/runtime.cc:289]   native: #11 pc 0039a56f  /system/lib/libart.so (art::ReferenceMapVisitor<art::RootCallbackVisitor>::VisitFrame()+319)
    art/runtime/runtime.cc:289]   native: #12 pc 00384c47  /system/lib/libart.so (art::StackVisitor::WalkStack(bool)+151)
    art/runtime/runtime.cc:289]   native: #13 pc 0038a340  /system/lib/libart.so (art::Thread::VisitRoots(void (*)(art::mirror::Object**, void*, art::RootInfo const&), void*)+1744)
    art/runtime/runtime.cc:289]   native: #14 pc 001cbca9  /system/lib/libart.so (art::gc::collector::CheckpointMarkThreadRoots::Run(art::Thread*)+137)
    art/runtime/runtime.cc:289]   native: #15 pc 0038ac29  /system/lib/libart.so (art::Thread::RunCheckpointFunction()+185)
    art/runtime/runtime.cc:289]   native: #16 pc 0042e4fc  /system/lib/libart.so (artTestSuspendFromCode+92)
    art/runtime/runtime.cc:289]   native: #17 pc 000e5b52  /system/lib/libart.so (art_quick_test_suspend+34)
    art/runtime/runtime.cc:289]   native: #18 pc 0018df8f  /data/dalvik-cache/x86/data@app@com.example.ujjwalgoyal.lottiesample-1@base.apk@classes.dex (Java_com_airbnb_lottie_animation_keyframe_BaseKeyframeAnimation_notifyListeners__+211)
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.animation.keyframe.BaseKeyframeAnimation.notifyListeners(BaseKeyframeAnimation.java:62)
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.animation.keyframe.BaseKeyframeAnimation.setProgress(BaseKeyframeAnimation.java:55)
02-22 14:42:12.002 5404-5404/com.example.ujjwalgoyal.lottiesample A/art: art/runtime/runtime.cc:289]   at com.airbnb.lottie.model.layer.BaseLayer.setProgress(BaseLayer.java:427)
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.model.layer.CompositionLayer.setProgress(CompositionLayer.java:139)
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.model.layer.CompositionLayer.setProgress(CompositionLayer.java:139)
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.model.layer.CompositionLayer.setProgress(CompositionLayer.java:139)
    art/runtime/runtime.cc:289]   ... repeated 1 times
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.LottieDrawable$1.onAnimationUpdate(LottieDrawable.java:103)
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.utils.BaseLottieAnimator.notifyUpdate(BaseLottieAnimator.java:88)
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.utils.LottieValueAnimator.doFrame(LottieValueAnimator.java:96)
    art/runtime/runtime.cc:289]   at android.view.Choreographer$CallbackRecord.run(Choreographer.java:765)
    art/runtime/runtime.cc:289]   at android.view.Choreographer.doCallbacks(Choreographer.java:580)
    art/runtime/runtime.cc:289]   at android.view.Choreographer.doFrame(Choreographer.java:549)
02-22 14:42:12.003 5404-5404/com.example.ujjwalgoyal.lottiesample A/art: art/runtime/runtime.cc:289]   at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:753)
    art/runtime/runtime.cc:289]   at android.os.Handler.handleCallback(Handler.java:739)
    art/runtime/runtime.cc:289]   at android.os.Handler.dispatchMessage(Handler.java:95)
    art/runtime/runtime.cc:289]   at android.os.Looper.loop(Looper.java:135)
    art/runtime/runtime.cc:289]   at android.app.ActivityThread.main(ActivityThread.java:5254)
    art/runtime/runtime.cc:289]   at java.lang.reflect.Method.invoke!(Native method)
    art/runtime/runtime.cc:289]   at java.lang.reflect.Method.invoke(Method.java:372)
    art/runtime/runtime.cc:289]   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
    art/runtime/runtime.cc:289]   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
    art/runtime/runtime.cc:289] Dumping all threads without appropriate locks held: thread list lock
    art/runtime/runtime.cc:289] All threads:
    art/runtime/runtime.cc:289] DALVIK THREADS (16):
    art/runtime/runtime.cc:289] "main" prio=5 tid=1 Runnable
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x7310f000 self=0xb4025800
    art/runtime/runtime.cc:289]   | sysTid=5404 nice=0 cgrp=default sched=0/0 handle=0xb77ddea0
    art/runtime/runtime.cc:289]   | state=R schedstat=( 0 0 0 ) utm=173 stm=16 core=1 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xbf3b6000-0xbf3b8000 stackSize=8MB
    art/runtime/runtime.cc:289]   | held mutexes= "abort lock" "mutator lock"(shared held)
    art/runtime/runtime.cc:289]   native: #00 pc 00005d03  /system/lib/libbacktrace_libc++.so (UnwindCurrent::Unwind(unsigned int, ucontext*)+83)
    art/runtime/runtime.cc:289]   native: #01 pc 00003051  /system/lib/libbacktrace_libc++.so (Backtrace::Unwind(unsigned int, ucontext*)+33)
    art/runtime/runtime.cc:289]   native: #02 pc 003c9ea7  /system/lib/libart.so (art::DumpNativeStack(std::__1::basic_ostream<char, std::__1::char_traits<char> >&, int, char const*, art::mirror::ArtMethod*)+135)
    art/runtime/runtime.cc:289]   native: #03 pc 00393212  /system/lib/libart.so (art::Thread::Dump(std::__1::basic_ostream<char, std::__1::char_traits<char> >&) const+306)
    art/runtime/runtime.cc:289]   native: #04 pc 0039f65e  /system/lib/libart.so (art::ThreadList::DumpLocked(std::__1::basic_ostream<char, std::__1::char_traits<char> >&)+478)
    art/runtime/runtime.cc:289]   native: #05 pc 003778bc  /system/lib/libart.so (art::AbortState::Dump(std::__1::basic_ostream<char, std::__1::char_traits<char> >&)+508)
02-22 14:42:12.004 5404-5404/com.example.ujjwalgoyal.lottiesample A/art: art/runtime/runtime.cc:289]   native: #06 pc 00377ead  /system/lib/libart.so (art::Runtime::Abort()+125)
    art/runtime/runtime.cc:289]   native: #07 pc 000ec704  /system/lib/libart.so (art::LogMessage::~LogMessage()+1876)
    art/runtime/runtime.cc:289]   native: #08 pc 001cb8d1  /system/lib/libart.so (bool art::gc::accounting::HeapBitmap::AtomicTestAndSet<art::gc::collector::MarkSweepMarkObjectSlowPath>(art::mirror::Object const*, art::gc::collector::MarkSweepMarkObjectSlowPath const&)+625)
    art/runtime/runtime.cc:289]   native: #09 pc 00084e4a  /system/lib/libart.so (art::gc::collector::MarkSweep::MarkObjectParallel(art::mirror::Object const*) (.part.121)+38)
    art/runtime/runtime.cc:289]   native: #10 pc 001cf74a  /system/lib/libart.so (art::gc::collector::MarkSweep::MarkRootParallelCallback(art::mirror::Object**, void*, art::RootInfo const&)+138)
    art/runtime/runtime.cc:289]   native: #11 pc 0039a015  /system/lib/libart.so (art::ReferenceMapVisitor<art::RootCallbackVisitor>::VisitQuickFrame()+1269)
    art/runtime/runtime.cc:289]   native: #12 pc 0039a56f  /system/lib/libart.so (art::ReferenceMapVisitor<art::RootCallbackVisitor>::VisitFrame()+319)
    art/runtime/runtime.cc:289]   native: #13 pc 00384c47  /system/lib/libart.so (art::StackVisitor::WalkStack(bool)+151)
    art/runtime/runtime.cc:289]   native: #14 pc 0038a340  /system/lib/libart.so (art::Thread::VisitRoots(void (*)(art::mirror::Object**, void*, art::RootInfo const&), void*)+1744)
    art/runtime/runtime.cc:289]   native: #15 pc 001cbca9  /system/lib/libart.so (art::gc::collector::CheckpointMarkThreadRoots::Run(art::Thread*)+137)
    art/runtime/runtime.cc:289]   native: #16 pc 0038ac29  /system/lib/libart.so (art::Thread::RunCheckpointFunction()+185)
    art/runtime/runtime.cc:289]   native: #17 pc 0042e4fc  /system/lib/libart.so (artTestSuspendFromCode+92)
    art/runtime/runtime.cc:289]   native: #18 pc 000e5b52  /system/lib/libart.so (art_quick_test_suspend+34)
    art/runtime/runtime.cc:289]   native: #19 pc 0018df8f  /data/dalvik-cache/x86/data@app@com.example.ujjwalgoyal.lottiesample-1@base.apk@classes.dex (Java_com_airbnb_lottie_animation_keyframe_BaseKeyframeAnimation_notifyListeners__+211)
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.animation.keyframe.BaseKeyframeAnimation.notifyListeners(BaseKeyframeAnimation.java:62)
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.animation.keyframe.BaseKeyframeAnimation.setProgress(BaseKeyframeAnimation.java:55)
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.model.layer.BaseLayer.setProgress(BaseLayer.java:427)
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.model.layer.CompositionLayer.setProgress(CompositionLayer.java:139)
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.model.layer.CompositionLayer.setProgress(CompositionLayer.java:139)
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.model.layer.CompositionLayer.setProgress(CompositionLayer.java:139)
    art/runtime/runtime.cc:289]   ... repeated 1 times
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.LottieDrawable$1.onAnimationUpdate(LottieDrawable.java:103)
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.utils.BaseLottieAnimator.notifyUpdate(BaseLottieAnimator.java:88)
    art/runtime/runtime.cc:289]   at com.airbnb.lottie.utils.LottieValueAnimator.doFrame(LottieValueAnimator.java:96)
    art/runtime/runtime.cc:289]   at android.view.Choreographer$CallbackRecord.run(Choreographer.java:765)
    art/runtime/runtime.cc:289]   at android.view.Choreographer.doCallbacks(Choreographer.java:580)
    art/runtime/runtime.cc:289]   at android.view.Choreographer.doFrame(Choreographer.java:549)
    art/runtime/runtime.cc:289]   at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:753)
    art/runtime/runtime.cc:289]   at android.os.Handler.handleCallback(Handler.java:739)
    art/runtime/runtime.cc:289]   at android.os.Handler.dispatchMessage(Handler.java:95)
    art/runtime/runtime.cc:289]   at android.os.Looper.loop(Looper.java:135)
    art/runtime/runtime.cc:289]   at android.app.ActivityThread.main(ActivityThread.java:5254)
    art/runtime/runtime.cc:289]   at java.lang.reflect.Method.invoke!(Native method)
    art/runtime/runtime.cc:289]   at java.lang.reflect.Method.invoke(Method.java:372)
    art/runtime/runtime.cc:289]   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
    art/runtime/runtime.cc:289]   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
    art/runtime/runtime.cc:289] 
    art/runtime/runtime.cc:289] "Heap thread pool worker thread 0" prio=5 tid=2 Native (still starting up)
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x0 self=0xae862400
    art/runtime/runtime.cc:289]   | sysTid=5407 nice=0 cgrp=default sched=0/0 handle=0xb4053100
    art/runtime/runtime.cc:289]   | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=3 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xb35f7000-0xb35f9000 stackSize=1020KB
    art/runtime/runtime.cc:289]   | held mutexes=
    art/runtime/runtime.cc:289]   kernel: futex_wait_queue_me+0xcd/0x113
    art/runtime/runtime.cc:289]   kernel: futex_wait+0xc5/0x197
    art/runtime/runtime.cc:289]   kernel: do_futex+0x9b/0x71e
    art/runtime/runtime.cc:289]   kernel: SyS_futex+0xaf/0xf6
    art/runtime/runtime.cc:289]   kernel: syscall_call+0x7/0xb
    art/runtime/runtime.cc:289]   native: #00 pc 000132d0  /system/lib/libc.so (syscall+32)
    art/runtime/runtime.cc:289]   native: #01 pc 000fdc97  /dev/ashmem/dalvik-Heap thread pool worker thread 0 (deleted) (???)
    art/runtime/runtime.cc:289]   (no managed stack frames)
    art/runtime/runtime.cc:289] 
    art/runtime/runtime.cc:289] "Heap thread pool worker thread 1" prio=5 tid=3 Native (still starting up)
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x0 self=0xb4025c00
    art/runtime/runtime.cc:289]   | sysTid=5408 nice=0 cgrp=default sched=0/0 handle=0xb4053b00
    art/runtime/runtime.cc:289]   | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=0 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xb34f7000-0xb34f9000 stackSize=1020KB
    art/runtime/runtime.cc:289]   | held mutexes=
    art/runtime/runtime.cc:289]   kernel: futex_wait_queue_me+0xcd/0x113
    art/runtime/runtime.cc:289]   kernel: futex_wait+0xc5/0x197
    art/runtime/runtime.cc:289]   kernel: do_futex+0x9b/0x71e
    art/runtime/runtime.cc:289]   kernel: SyS_futex+0xaf/0xf6
    art/runtime/runtime.cc:289]   kernel: syscall_call+0x7/0xb
    art/runtime/runtime.cc:289]   native: #00 pc 000132d0  /system/lib/libc.so (syscall+32)
    art/runtime/runtime.cc:289]   native: #01 pc 000fdc97  /dev/ashmem/dalvik-Heap thread pool worker thread 1 (deleted) (???)
    art/runtime/runtime.cc:289]   (no managed stack frames)
    art/runtime/runtime.cc:289] 
    art/runtime/runtime.cc:289] "Heap thread pool worker thread 2" prio=5 tid=4 Native (still starting up)
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x0 self=0xae863400
    art/runtime/runtime.cc:289]   | sysTid=5409 nice=0 cgrp=default sched=0/0 handle=0xb4053d80
    art/runtime/runtime.cc:289]   | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=2 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xb33f7000-0xb33f9000 stackSize=1020KB
    art/runtime/runtime.cc:289]   | held mutexes=
    art/runtime/runtime.cc:289]   kernel: futex_wait_queue_me+0xcd/0x113
    art/runtime/runtime.cc:289]   kernel: futex_wait+0xc5/0x197
    art/runtime/runtime.cc:289]   kernel: do_futex+0x9b/0x71e
    art/runtime/runtime.cc:289]   kernel: SyS_futex+0xaf/0xf6
    art/runtime/runtime.cc:289]   kernel: syscall_call+0x7/0xb
    art/runtime/runtime.cc:289]   native: #00 pc 000132d0  /system/lib/libc.so (syscall+32)
    art/runtime/runtime.cc:289]   native: #01 pc 000fdc97  /dev/ashmem/dalvik-Heap thread pool worker thread 2 (deleted) (???)
    art/runtime/runtime.cc:289]   (no managed stack frames)
    art/runtime/runtime.cc:289] 
    art/runtime/runtime.cc:289] "Signal Catcher" prio=5 tid=5 WaitingInMainSignalCatcherLoop
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x12c000a0 self=0xb4030c00
    art/runtime/runtime.cc:289]   | sysTid=5411 nice=0 cgrp=default sched=0/0 handle=0xb4051580
    art/runtime/runtime.cc:289]   | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=3 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xb32f9000-0xb32fb000 stackSize=1012KB
02-22 14:42:12.005 5404-5404/com.example.ujjwalgoyal.lottiesample A/art: art/runtime/runtime.cc:289]   | held mutexes=
    art/runtime/runtime.cc:289]   kernel: do_sigtimedwait+0xd7/0x19f
    art/runtime/runtime.cc:289]   kernel: SyS_rt_sigtimedwait+0x6d/0x9c
    art/runtime/runtime.cc:289]   kernel: syscall_call+0x7/0xb
    art/runtime/runtime.cc:289]   native: #00 pc 00075ceb  /system/lib/libc.so (__rt_sigtimedwait+27)
    art/runtime/runtime.cc:289]   native: #01 pc 00024367  /system/lib/libc.so (sigwait+71)
    art/runtime/runtime.cc:289]   native: #02 pc 00380015  /system/lib/libart.so (art::SignalCatcher::WaitForSignal(art::Thread*, art::SignalSet&)+133)
    art/runtime/runtime.cc:289]   native: #03 pc 00380c32  /system/lib/libart.so (art::SignalCatcher::Run(void*)+306)
    art/runtime/runtime.cc:289]   native: #04 pc 000211a8  /system/lib/libc.so (__pthread_start(void*)+56)
    art/runtime/runtime.cc:289]   native: #05 pc 0001c529  /system/lib/libc.so (__start_thread+25)
    art/runtime/runtime.cc:289]   native: #06 pc 000130f6  /system/lib/libc.so (__bionic_clone+70)
    art/runtime/runtime.cc:289]   (no managed stack frames)
    art/runtime/runtime.cc:289] 
    art/runtime/runtime.cc:289] "JDWP" prio=5 tid=6 WaitingInMainDebuggerLoop
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x12c050a0 self=0xae864400
    art/runtime/runtime.cc:289]   | sysTid=5412 nice=0 cgrp=default sched=0/0 handle=0xb4051800
    art/runtime/runtime.cc:289]   | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=3 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xb31fb000-0xb31fd000 stackSize=1012KB
    art/runtime/runtime.cc:289]   | held mutexes=
    art/runtime/runtime.cc:289]   kernel: poll_schedule_timeout+0x42/0x8e
    art/runtime/runtime.cc:289]   kernel: do_select+0x4a6/0x4f4
    art/runtime/runtime.cc:289]   kernel: core_sys_select+0x16c/0x24c
    art/runtime/runtime.cc:289]   kernel: SyS_pselect6+0xff/0x17a
    art/runtime/runtime.cc:289]   kernel: syscall_call+0x7/0xb
    art/runtime/runtime.cc:289]   native: #00 pc 00074df5  /system/lib/libc.so (__pselect6+37)
    art/runtime/runtime.cc:289]   native: #01 pc 0001fcc3  /system/lib/libc.so (select+179)
    art/runtime/runtime.cc:289]   native: #02 pc 00439f3b  /system/lib/libart.so (art::JDWP::JdwpAdbState::ProcessIncoming()+379)
    art/runtime/runtime.cc:289]   native: #03 pc 0028f7af  /system/lib/libart.so (art::JDWP::JdwpState::Run()+415)
    art/runtime/runtime.cc:289]   native: #04 pc 002918a0  /system/lib/libart.so (art::JDWP::StartJdwpThread(void*)+32)
    art/runtime/runtime.cc:289]   native: #05 pc 000211a8  /system/lib/libc.so (__pthread_start(void*)+56)
    art/runtime/runtime.cc:289]   native: #06 pc 0001c529  /system/lib/libc.so (__start_thread+25)
    art/runtime/runtime.cc:289]   native: #07 pc 000130f6  /system/lib/libc.so (__bionic_clone+70)
    art/runtime/runtime.cc:289]   (no managed stack frames)
    art/runtime/runtime.cc:289] 
    art/runtime/runtime.cc:289] "ReferenceQueueDaemon" prio=5 tid=7 Waiting
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x12c060a0 self=0xb4026400
    art/runtime/runtime.cc:289]   | sysTid=5415 nice=0 cgrp=default sched=0/0 handle=0xb4051a80
    art/runtime/runtime.cc:289]   | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=2 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xb30f7000-0xb30f9000 stackSize=1036KB
    art/runtime/runtime.cc:289]   | held mutexes=
    art/runtime/runtime.cc:289]   kernel: futex_wait_queue_me+0xcd/0x113
    art/runtime/runtime.cc:289]   kernel: futex_wait+0xc5/0x197
    art/runtime/runtime.cc:289]   kernel: do_futex+0x9b/0x71e
    art/runtime/runtime.cc:289]   kernel: SyS_futex+0xaf/0xf6
    art/runtime/runtime.cc:289]   kernel: syscall_call+0x7/0xb
    art/runtime/runtime.cc:289]   native: #00 pc 000132d0  /system/lib/libc.so (syscall+32)
    art/runtime/runtime.cc:289]   native: #01 pc 001018e7  [stack:5415] (???)
    art/runtime/runtime.cc:289]   at java.lang.Object.wait!(Native method)
    art/runtime/runtime.cc:289]   - waiting on <0x1942ec13> (a java.lang.Class<java.lang.ref.ReferenceQueue>)
    art/runtime/runtime.cc:289]   at java.lang.Daemons$ReferenceQueueDaemon.run(Daemons.java:134)
    art/runtime/runtime.cc:289]   - locked <0x1942ec13> (a java.lang.Class<java.lang.ref.ReferenceQueue>)
    art/runtime/runtime.cc:289]   at java.lang.Thread.run(Thread.java:818)
    art/runtime/runtime.cc:289] 
    art/runtime/runtime.cc:289] "FinalizerDaemon" prio=5 tid=8 Waiting
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x12c06100 self=0xb4026800
    art/runtime/runtime.cc:289]   | sysTid=5416 nice=0 cgrp=default sched=0/0 handle=0xb4051d00
    art/runtime/runtime.cc:289]   | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=0 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xb2ff3000-0xb2ff5000 stackSize=1036KB
    art/runtime/runtime.cc:289]   | held mutexes=
    art/runtime/runtime.cc:289]   kernel: futex_wait_queue_me+0xcd/0x113
    art/runtime/runtime.cc:289]   kernel: futex_wait+0xc5/0x197
    art/runtime/runtime.cc:289]   kernel: do_futex+0x9b/0x71e
    art/runtime/runtime.cc:289]   kernel: SyS_futex+0xaf/0xf6
    art/runtime/runtime.cc:289]   kernel: syscall_call+0x7/0xb
    art/runtime/runtime.cc:289]   native: #00 pc 000132d0  /system/lib/libc.so (syscall+32)
    art/runtime/runtime.cc:289]   native: #01 pc 00101807  [stack:5416] (???)
    art/runtime/runtime.cc:289]   at java.lang.Object.wait!(Native method)
    art/runtime/runtime.cc:289]   - waiting on <0x0f1e7e50> (a java.lang.ref.ReferenceQueue)
    art/runtime/runtime.cc:289]   at java.lang.Object.wait(Object.java:422)
    art/runtime/runtime.cc:289]   at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:101)
    art/runtime/runtime.cc:289]   - locked <0x0f1e7e50> (a java.lang.ref.ReferenceQueue)
    art/runtime/runtime.cc:289]   at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:72)
    art/runtime/runtime.cc:289]   at java.lang.Daemons$FinalizerDaemon.run(Daemons.java:174)
    art/runtime/runtime.cc:289]   at java.lang.Thread.run(Thread.java:818)
    art/runtime/runtime.cc:289] 
    art/runtime/runtime.cc:289] "FinalizerWatchdogDaemon" prio=5 tid=9 Sleeping
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x12c06160 self=0xb4031c00
    art/runtime/runtime.cc:289]   | sysTid=5417 nice=0 cgrp=default sched=0/0 handle=0xb4051f80
    art/runtime/runtime.cc:289]   | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=0 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xb2eef000-0xb2ef1000 stackSize=1036KB
    art/runtime/runtime.cc:289]   | held mutexes=
    art/runtime/runtime.cc:289]   kernel: futex_wait_queue_me+0xcd/0x113
    art/runtime/runtime.cc:289]   kernel: futex_wait+0xc5/0x197
    art/runtime/runtime.cc:289]   kernel: do_futex+0x9b/0x71e
    art/runtime/runtime.cc:289]   kernel: SyS_futex+0xaf/0xf6
    art/runtime/runtime.cc:289]   kernel: syscall_call+0x7/0xb
    art/runtime/runtime.cc:289]   native: #00 pc 000132d2  /system/lib/libc.so (syscall+34)
    art/runtime/runtime.cc:289]   native: #01 pc 00101797  [stack:5417] (???)
    art/runtime/runtime.cc:289]   at java.lang.Thread.sleep!(Native method)
    art/runtime/runtime.cc:289]   - sleeping on <0x3c585849> (a java.lang.Object)
    art/runtime/runtime.cc:289]   at java.lang.Thread.sleep(Thread.java:1031)
    art/runtime/runtime.cc:289]   - locked <0x3c585849> (a java.lang.Object)
    art/runtime/runtime.cc:289]   at java.lang.Thread.sleep(Thread.java:985)
    art/runtime/runtime.cc:289]   at java.lang.Daemons$FinalizerWatchdogDaemon.sleepFor(Daemons.java:258)
    art/runtime/runtime.cc:289]   at java.lang.Daemons$FinalizerWatchdogDaemon.waitForFinalization(Daemons.java:269)
    art/runtime/runtime.cc:289]   at java.lang.Daemons$FinalizerWatchdogDaemon.run(Daemons.java:217)
    art/runtime/runtime.cc:289]   at java.lang.Thread.run(Thread.java:818)
    art/runtime/runtime.cc:289] 
    art/runtime/runtime.cc:289] "HeapTrimmerDaemon" prio=5 tid=10 Sleeping
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x12c061c0 self=0xb4032000
    art/runtime/runtime.cc:289]   | sysTid=5418 nice=0 cgrp=default sched=0/0 handle=0xb4054280
    art/runtime/runtime.cc:289]   | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=3 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xb2deb000-0xb2ded000 stackSize=1036KB
    art/runtime/runtime.cc:289]   | held mutexes=
    art/runtime/runtime.cc:289]   kernel: hrtimer_nanosleep+0x75/0xd2
    art/runtime/runtime.cc:289]   kernel: SyS_nanosleep+0x45/0x4f
    art/runtime/runtime.cc:289]   kernel: syscall_call+0x7/0xb
    art/runtime/runtime.cc:289]   native: #00 pc 00074f21  /system/lib/libc.so (nanosleep+17)
    art/runtime/runtime.cc:289]   native: #01 pc 000433f8  /system/lib/libc.so (usleep+72)
    art/runtime/runtime.cc:289]   native: #02 pc 001faa2c  /system/lib/libart.so (art::gc::Heap::DoPendingTransitionOrTrim()+1052)
    art/runtime/runtime.cc:289]   native: #03 pc 00337b54  /system/lib/libart.so (art::VMRuntime_trimHeap(_JNIEnv*, _jobject*)+36)
    art/runtime/runtime.cc:289]   native: #04 pc 000004ef  /data/dalvik-cache/x86/system@framework@boot.oat (Java_dalvik_system_VMRuntime_trimHeap__+115)
    art/runtime/runtime.cc:289]   at dalvik.system.VMRuntime.trimHeap(Native method)
    art/runtime/runtime.cc:289]   - sleeping on an unknown object
    art/runtime/runtime.cc:289]   at java.lang.Daemons$HeapTrimmerDaemon.run(Daemons.java:313)
    art/runtime/runtime.cc:289]   at java.lang.Thread.run(Thread.java:818)
    art/runtime/runtime.cc:289] 
    art/runtime/runtime.cc:289] "GCDaemon" prio=5 tid=11 WaitingForCheckPointsToRun
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x12c06220 self=0xb4032400
    art/runtime/runtime.cc:289]   | sysTid=5419 nice=0 cgrp=default sched=0/0 handle=0xb4054500
    art/runtime/runtime.cc:289]   | state=S schedstat=( 0 0 0 ) utm=15 stm=4 core=2 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xaef2b000-0xaef2d000 stackSize=1036KB
    art/runtime/runtime.cc:289]   | held mutexes=
    art/runtime/runtime.cc:289]   kernel: futex_wait_queue_me+0xcd/0x113
    art/runtime/runtime.cc:289]   kernel: futex_wait+0xc5/0x197
02-22 14:42:12.006 5404-5404/com.example.ujjwalgoyal.lottiesample A/art: art/runtime/runtime.cc:289]   kernel: do_futex+0x9b/0x71e
    art/runtime/runtime.cc:289]   kernel: SyS_futex+0xaf/0xf6
    art/runtime/runtime.cc:289]   kernel: syscall_call+0x7/0xb
    art/runtime/runtime.cc:289]   native: #00 pc 000132d0  /system/lib/libc.so (syscall+32)
    art/runtime/runtime.cc:289]   native: #01 pc 001011b7  [stack:5419] (???)
    art/runtime/runtime.cc:289]   at dalvik.system.VMRuntime.concurrentGC(Native method)
    art/runtime/runtime.cc:289]   at java.lang.Daemons$GCDaemon.run(Daemons.java:346)
    art/runtime/runtime.cc:289]   at java.lang.Thread.run(Thread.java:818)
    art/runtime/runtime.cc:289] 
    art/runtime/runtime.cc:289] "Binder_1" prio=5 tid=12 Native
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x12c570a0 self=0xae865400
    art/runtime/runtime.cc:289]   | sysTid=5420 nice=0 cgrp=default sched=0/0 handle=0xb4054780
    art/runtime/runtime.cc:289]   | state=S schedstat=( 0 0 0 ) utm=0 stm=1 core=0 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xaed2f000-0xaed31000 stackSize=1012KB
    art/runtime/runtime.cc:289]   | held mutexes=
    art/runtime/runtime.cc:289]   kernel: binder_thread_read+0xb98/0xbc4
    art/runtime/runtime.cc:289]   kernel: binder_ioctl+0x2fd/0x60f
    art/runtime/runtime.cc:289]   kernel: do_vfs_ioctl+0x3ab/0x454
    art/runtime/runtime.cc:289]   kernel: SyS_ioctl+0x4a/0x6a
    art/runtime/runtime.cc:289]   kernel: syscall_call+0x7/0xb
    art/runtime/runtime.cc:289]   native: #00 pc 00074324  /system/lib/libc.so (__ioctl+20)
    art/runtime/runtime.cc:289]   native: #01 pc 000ad8ca  /system/lib/libc.so (ioctl+42)
    art/runtime/runtime.cc:289]   native: #02 pc 0002bc1b  /system/lib/libbinder.so (android::IPCThreadState::talkWithDriver(bool)+203)
    art/runtime/runtime.cc:289]   native: #03 pc 0002c575  /system/lib/libbinder.so (android::IPCThreadState::getAndExecuteCommand()+37)
    art/runtime/runtime.cc:289]   native: #04 pc 0002c639  /system/lib/libbinder.so (android::IPCThreadState::joinThreadPool(bool)+105)
    art/runtime/runtime.cc:289]   native: #05 pc 00034ea8  /system/lib/libbinder.so (android::PoolThread::threadLoop()+40)
    art/runtime/runtime.cc:289]   native: #06 pc 00016a5e  /system/lib/libutils.so (android::Thread::_threadLoop(void*)+398)
    art/runtime/runtime.cc:289]   native: #07 pc 00071ca2  /system/lib/libandroid_runtime.so (android::AndroidRuntime::javaThreadShell(void*)+98)
    art/runtime/runtime.cc:289]   native: #08 pc 0001617a  /system/lib/libutils.so (thread_data_t::trampoline(thread_data_t const*)+122)
    art/runtime/runtime.cc:289]   native: #09 pc 000211a8  /system/lib/libc.so (__pthread_start(void*)+56)
    art/runtime/runtime.cc:289]   native: #10 pc 0001c529  /system/lib/libc.so (__start_thread+25)
    art/runtime/runtime.cc:289]   native: #11 pc 000130f6  /system/lib/libc.so (__bionic_clone+70)
    art/runtime/runtime.cc:289]   (no managed stack frames)
    art/runtime/runtime.cc:289] 
    art/runtime/runtime.cc:289] "Binder_2" prio=5 tid=13 Native
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x12c5a0a0 self=0xb4032c00
    art/runtime/runtime.cc:289]   | sysTid=5421 nice=0 cgrp=default sched=0/0 handle=0xae8a9080
    art/runtime/runtime.cc:289]   | state=S schedstat=( 0 0 0 ) utm=0 stm=1 core=2 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xaec31000-0xaec33000 stackSize=1012KB
    art/runtime/runtime.cc:289]   | held mutexes=
    art/runtime/runtime.cc:289]   kernel: binder_thread_read+0xb98/0xbc4
    art/runtime/runtime.cc:289]   kernel: binder_ioctl+0x2fd/0x60f
    art/runtime/runtime.cc:289]   kernel: do_vfs_ioctl+0x3ab/0x454
    art/runtime/runtime.cc:289]   kernel: SyS_ioctl+0x4a/0x6a
    art/runtime/runtime.cc:289]   kernel: syscall_call+0x7/0xb
    art/runtime/runtime.cc:289]   native: #00 pc 00074324  /system/lib/libc.so (__ioctl+20)
    art/runtime/runtime.cc:289]   native: #01 pc 000ad8ca  /system/lib/libc.so (ioctl+42)
    art/runtime/runtime.cc:289]   native: #02 pc 0002bc1b  /system/lib/libbinder.so (android::IPCThreadState::talkWithDriver(bool)+203)
    art/runtime/runtime.cc:289]   native: #03 pc 0002c575  /system/lib/libbinder.so (android::IPCThreadState::getAndExecuteCommand()+37)
    art/runtime/runtime.cc:289]   native: #04 pc 0002c639  /system/lib/libbinder.so (android::IPCThreadState::joinThreadPool(bool)+105)
    art/runtime/runtime.cc:289]   native: #05 pc 00034ea8  /system/lib/libbinder.so (android::PoolThread::threadLoop()+40)
    art/runtime/runtime.cc:289]   native: #06 pc 00016a5e  /system/lib/libutils.so (android::Thread::_threadLoop(void*)+398)
    art/runtime/runtime.cc:289]   native: #07 pc 00071ca2  /system/lib/libandroid_runtime.so (android::AndroidRuntime::javaThreadShell(void*)+98)
    art/runtime/runtime.cc:289]   native: #08 pc 0001617a  /system/lib/libutils.so (thread_data_t::trampoline(thread_data_t const*)+122)
    art/runtime/runtime.cc:289]   native: #09 pc 000211a8  /system/lib/libc.so (__pthread_start(void*)+56)
    art/runtime/runtime.cc:289]   native: #10 pc 0001c529  /system/lib/libc.so (__start_thread+25)
    art/runtime/runtime.cc:289]   native: #11 pc 000130f6  /system/lib/libc.so (__bionic_clone+70)
    art/runtime/runtime.cc:289]   (no managed stack frames)
    art/runtime/runtime.cc:289] 
    art/runtime/runtime.cc:289] "pool-1-thread-1" prio=5 tid=14 TimedWaiting
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x12c00ca0 self=0xb43af400
    art/runtime/runtime.cc:289]   | sysTid=5423 nice=0 cgrp=default sched=0/0 handle=0xb4054a00
    art/runtime/runtime.cc:289]   | state=S schedstat=( 0 0 0 ) utm=2 stm=0 core=0 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xae6fe000-0xae700000 stackSize=1036KB
    art/runtime/runtime.cc:289]   | held mutexes=
    art/runtime/runtime.cc:289]   kernel: futex_wait_queue_me+0xcd/0x113
    art/runtime/runtime.cc:289]   kernel: futex_wait+0xc5/0x197
    art/runtime/runtime.cc:289]   kernel: do_futex+0x9b/0x71e
    art/runtime/runtime.cc:289]   kernel: SyS_futex+0xaf/0xf6
    art/runtime/runtime.cc:289]   kernel: syscall_call+0x7/0xb
    art/runtime/runtime.cc:289]   native: #00 pc 000132d2  /system/lib/libc.so (syscall+34)
    art/runtime/runtime.cc:289]   native: #01 pc 00101667  [stack:5423] (???)
    art/runtime/runtime.cc:289]   at java.lang.Object.wait!(Native method)
    art/runtime/runtime.cc:289]   - waiting on <0x2dfa054e> (a java.lang.Object)
    art/runtime/runtime.cc:289]   at java.lang.Thread.parkFor(Thread.java:1220)
    art/runtime/runtime.cc:289]   - locked <0x2dfa054e> (a java.lang.Object)
    art/runtime/runtime.cc:289]   at sun.misc.Unsafe.park(Unsafe.java:299)
    art/runtime/runtime.cc:289]   at java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:197)
    art/runtime/runtime.cc:289]   at java.util.concurrent.SynchronousQueue$TransferStack.awaitFulfill(SynchronousQueue.java:429)
    art/runtime/runtime.cc:289]   at java.util.concurrent.SynchronousQueue$TransferStack.transfer(SynchronousQueue.java:331)
    art/runtime/runtime.cc:289]   at java.util.concurrent.SynchronousQueue.poll(SynchronousQueue.java:910)
    art/runtime/runtime.cc:289]   at java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1035)
    art/runtime/runtime.cc:289]   at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1097)
    art/runtime/runtime.cc:289]   at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
    art/runtime/runtime.cc:289]   at java.lang.Thread.run(Thread.java:818)
    art/runtime/runtime.cc:289] 
    art/runtime/runtime.cc:289] "RenderThread" prio=5 tid=16 Native
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x12c06280 self=0xae866400
    art/runtime/runtime.cc:289]   | sysTid=5425 nice=-4 cgrp=default sched=0/0 handle=0xb4055e00
    art/runtime/runtime.cc:289]   | state=S schedstat=( 0 0 0 ) utm=6 stm=10 core=3 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xa3e82000-0xa3e84000 stackSize=1012KB
    art/runtime/runtime.cc:289]   | held mutexes=
    art/runtime/runtime.cc:289]   kernel: SyS_epoll_wait+0x236/0x2d9
    art/runtime/runtime.cc:289]   kernel: SyS_epoll_pwait+0x70/0xe1
    art/runtime/runtime.cc:289]   kernel: syscall_call+0x7/0xb
    art/runtime/runtime.cc:289]   native: #00 pc 00075c25  /system/lib/libc.so (__epoll_pwait+37)
    art/runtime/runtime.cc:289]   native: #01 pc 0001cc72  /system/lib/libc.so (epoll_pwait+98)
    art/runtime/runtime.cc:289]   native: #02 pc 0001ccba  /system/lib/libc.so (epoll_wait+58)
    art/runtime/runtime.cc:289]   native: #03 pc 0001b857  /system/lib/libutils.so (android::Looper::pollInner(int)+199)
    art/runtime/runtime.cc:289]   native: #04 pc 0001bc57  /system/lib/libutils.so (android::Looper::pollOnce(int, int*, int*, void**)+55)
    art/runtime/runtime.cc:289]   native: #05 pc 00089ca4  /system/lib/libhwui.so (android::uirenderer::renderthread::RenderThread::threadLoop()+116)
    art/runtime/runtime.cc:289]   native: #06 pc 00016a5e  /system/lib/libutils.so (android::Thread::_threadLoop(void*)+398)
    art/runtime/runtime.cc:289]   native: #07 pc 00071ca2  /system/lib/libandroid_runtime.so (android::AndroidRuntime::javaThreadShell(void*)+98)
    art/runtime/runtime.cc:289]   native: #08 pc 0001617a  /system/lib/libutils.so (thread_data_t::trampoline(thread_data_t const*)+122)
    art/runtime/runtime.cc:289]   native: #09 pc 000211a8  /system/lib/libc.so (__pthread_start(void*)+56)
    art/runtime/runtime.cc:289]   native: #10 pc 0001c529  /system/lib/libc.so (__start_thread+25)
    art/runtime/runtime.cc:289]   native: #11 pc 000130f6  /system/lib/libc.so (__bionic_clone+70)
    art/runtime/runtime.cc:289]   (no managed stack frames)
    art/runtime/runtime.cc:289] 
    art/runtime/runtime.cc:289] "hwuiTask1" prio=5 tid=15 Native
    art/runtime/runtime.cc:289]   | group="" sCount=0 dsCount=0 obj=0x12c051c0 self=0xb43b1000
    art/runtime/runtime.cc:289]   | sysTid=5426 nice=-2 cgrp=default sched=0/0 handle=0xae8a9d00
    art/runtime/runtime.cc:289]   | state=S schedstat=( 0 0 0 ) utm=0 stm=1 core=3 HZ=100
    art/runtime/runtime.cc:289]   | stack=0xa3f86000-0xa3f88000 stackSize=1012KB
    art/runtime/runtime.cc:289]   | held mutexes=
    art/runtime/runtime.cc:289]   kernel: futex_wait_queue_me+0xcd/0x113
    art/runtime/runtime.cc:289]   kernel: futex_wait+0xc5/0x197
    art/runtime/runtime.cc:289]   kernel: do_futex+0x9b/0x71e
    art/runtime/runtime.cc:289]   kernel: SyS_futex+0xaf/0xf6
    art/runtime/runtime.cc:289]   kernel: syscall_call+0x7/0xb
    art/runtime/runtime.cc:289]   native: #00 pc 000132d0  /system/lib/libc.so (syscall+32)
    art/runtime/runtime.cc:289]   native: #01 pc 000fbdb7  [stack:5426] (???)
    art/runtime/runtime.cc:289]   (no managed stack frames)
    art/runtime/runtime.cc:289] 
    art/runtime/runtime.cc:289] 
02-22 14:42:12.006 5404-5404/com.example.ujjwalgoyal.lottiesample A/libc: Fatal signal 6 (SIGABRT), code -6 in tid 5404 (al.lottiesample)`
[end_transition.zip](https://github.com/airbnb/lottie-android/files/2893291/end_transition.zip)

**Comment by gpeal** [comment]
Duplicate of #1000

**Comment by gpeal** [comment]
@ujjwalnitw Does this happen 100% of the time for you?

**Comment by ujjwalnitw** [comment]
@gpeal 
Yes, it is 100% reproducible.

**Comment by ujjwalnitw** [comment]
@gpeal Anu update regarding this issue?

**Comment by gpeal** [comment]
@ujjwalnitw I can't get this to reproduce on any of my devices or emulators. Any assistance in repro steps is welcome.

**Comment by AndreiNikitin** [comment]
use 22 API emulator with x86 image (not x86_64). Open the screen with animation and wait for 2-3 min. Will crash

**Comment by AndreiNikitin** [comment]
happens on 3.0.0 version also

**Comment by AndreiNikitin** [comment]
![Screenshot 2019-07-26 at 11 37 59](https://user-images.githubusercontent.com/18249799/61938723-6b357280-af9a-11e9-8e12-5f5d6699aa71.png)
Here is a thread stack, if it helps you.

**Comment by AndreiNikitin** [comment]
@gpeal any news?
